examples/vanilla/control-elements/media-captions-selectmenu.html

```diff
@@ -71,11 +73,12 @@ <h2>Disabled</h2>
       muted
       crossorigin
     >
-      <track label="English" kind="captions" srclang="en" src="../vtt/elephantsdream/captions.en.vtt"></track>
+      <track label="German" kind="captions" srclang="de" src="../vtt/elephantsdream/captions.de.vtt"></track>
```

Comment:
Note: Changed this order to ensure we select navigator lang by default (instead of simply the first one).

This only applies if the local storage pref is disabled or if nothing has been stored to the local storage for the user pref yet.